aep/general/0133/aep.md.j2

```diff
@@ -0,0 +1,246 @@
+  have [strong consistency][]: the completion of a create operation **must**
+  mean that all user-settable values and the existence of the resource have
+  reached a steady-state and reading resource state returns a consistent
+  response.
+
+{% tab proto %}
+
+```proto
+rpc CreateBook(CreateBookRequest) returns (Book) {
+  option (google.api.http) = {
+    post: "/v1/{parent=publishers/*}/books"
+    body: "book"
+  };
+  option (google.api.method_signature) = "parent,book";
+}
+```
+
+- The RPC's name **must** begin with the word `Create`. The remainder of the
+  RPC name **should** be the singular form of the resource being created.
+  - The request message **must** match the RPC name, with a `Request` suffix.
+- The collection's parent resource **should** be called `parent`, and
+  **should** be the only variable in the URI path.
+  - The collection identifier (`books` in the above example) **must** be a
+    literal string.
+- There **must** be a `body` key in the `google.api.http` annotation, and it
+  **must** map to the resource field in the request message.
+  - All remaining fields **should** map to URI query parameters.
+- There **should** be exactly one `google.api.method_signature` annotation,
+  with a value of `"parent,{resource},id"`, or "`"parent,{resource}"` if the
+  resource ID is not required.
+
+{% tab oas %}
+
+**Note:** OAS guidance not yet written
+
+{% endtabs %}
+
+### Request message
+
+Create methods implement a common request message pattern:
+
+- An `id` field **must** be included for management plane resources, and
+  **should** be included for data plane resources.
+- The resource field **must** be included and **must** map to the POST body.
+- The request message **must not** contain any other required fields and
+  **should not** contain other optional fields except those described in this
+  or another AEP.
+
+{% tab proto %}
+
+```proto
+message CreateBookRequest {
+  // The parent resource where this book will be created.
+  // Format: publishers/{publisher}
+  string parent = 1 [
+    (google.api.field_behavior) = REQUIRED,
+    (google.api.resource_reference) = {
+      child_type: "library.example.com/Book"
+    }];
+
+  // The ID to use for the book, which will become the final component of
+  // the book's resource path.
+  //
+  // This value should be 4-63 characters, and valid characters
+  // are /[a-z][0-9]-/.
+  string id = 2 [(google.api.field_behavior) = REQUIRED];
+
+  // The book to create.
+  Book book = 3 [(google.api.field_behavior) = REQUIRED];
+}
+```
+
+- A `parent` field **must** be included unless the resource being created is a
+  top-level resource. It **should** be called `parent`.
+  - The field **should** be [annotated as `REQUIRED`][aep-203].
+  - The field **must** identify the [resource type][aep-123] of the resource
+    being created.
+
+{% tab oas %}
+
+**Note:** OAS guidance not yet written
+
+{% endtabs %}
+
+### Long-running create
```

Comment:
do we need this guidance? it's basically saying the same thing that AEP-151 says: if the request takes a long time, then use an LRO.

More duplicative guidance makes things harder to read, IMO.

Reply:
Thanks for filing https://github.com/aep-dev/aep.dev/issues/106 - assuming we can potentially clean this up later and not block the PR on it.

Reply:
I would have preferred this would be cleaned up in the future since you're touching the file already :P - but yes it's not a blocker.